cmd.go

```diff
@@ -153,6 +153,26 @@ func NewCmdOptions(options Options, name string, args ...string) *Cmd {
 	return out
 }
 
+// Clone clones a Cmd. All the options are transferred,
+// but the state of the original object is lost.
+// Cmd is one-use only, so if you need to re-start a Cmd,
+// you need to Clone it.
+func (c *Cmd) Clone() *Cmd {
+	clone := NewCmdOptions(
+		Options{
+			Buffered:  c.buffered,
+			Streaming: c.Stdout != nil,
+		},
+		c.Name,
+		c.Args...,
+	)
+	clone.Lock()
+	defer clone.Unlock()
```

Comment:
Why lock? `clone` hasn't been returned yet, so nothing else can be using it.

Reply:
Let me try. I remember testing with "-race" in similar cases and the tests were randomly failing because if race conditions.

Reply:
Ok, I removed the lock.